spec/citylist-spec.js

```diff
@@ -17,9 +17,13 @@ describe('CityList', function() {
 
         it('should contain only city objects', function() {
             var cities = JSON.parse(fs.readFileSync('cities/cities.json')),
-                city;
-            for (var i = 0, n = cities.length; i < n; i++) {
-                city = cities[i];
+                keys = Object.keys(cities),
+                city,
+                key;
+            for (var i = 0, n = keys.length; i < n; i++) {
+                key = keys[i];
+                city = cities[key];
+                expect(Object.keys(city).length).toBe(2);
```

Comment:
If the `label` is missing this correctly outputs `Expected 1 to be 2.`.
It would be great if the error message would be a bit more verbose.
Do you see any way to output the name of the missing key?

Reply:
Oh right, I missed that. Have moved the test below the two tests of the required properties, only checking that there are not more than two keys.

I also added messages for the checks for id and label (for better messages than "Expected undefined to be defined").
We could also add [jasmine-expect](https://www.npmjs.com/package/jasmine-expect) to make the test itself explicit: `expect(city).toHaveString('id')`. I have the code ready if you like the idea.

Reply:
Thank you. I will take a look soon. I will let you know about `jasmine-expect`, too. Sounds interesting.